doc/sphinx/source/using.rst

```diff
@@ -407,6 +407,31 @@ pocl.
  When set to 1, prints out remarks produced by the loop vectorizer of LLVM
  during kernel compilation.
 
+- **POCL_FORCE_VECTOR_WIDTH**
+
+ Forces the LLVM loop vectorizer to use the specified vector width (expressed
+ as a number of **loop iterations**), overriding the default value determined
+ by the vectorizer's cost model.
+ The same vector width will be used by all loops in all kernels.
+ Setting the vector width to 1 disables vectorization.
+ If the requested vector width is higher than the machine's native vector
+ width, the vectorizer will also unroll the loop.
+
+- **POCL_PREFER_VECTOR_WIDTH**
+
+ Override the preferred vector width (expressed as a number of **bits**) for
+ x86 targets.
```

Comment:
Is this x86-specific or might it work for other targets such as ARM or RISC-V?

Reply:
My understanding is that it's x86-specific.  I've only ever seen the prefer-vector-width=256 attribute being set for x86 targets, and only for the subset of x86 architectures where the clang/gcc community decided that 256-bit was a better default.